unit_tests/testAndersonAcceleration/testAndersonAcceleration.f90

```diff
@@ -47,6 +49,7 @@ PROGRAM testAndersonAcceleration
 
 CREATE_TEST('Test Anderson Acceleration Solver')
 
+CALL NativeRNG%init(1_SIK)
```

Comment:
One small thing: I think there are enumerations for this... should use the enumeration instead of a hard-coded integer.  Here and at line 68

Reply:
Thanks for fixing that incorrect example too

Reply:
Yeah, no problem.